pygmt/accessors.py

```diff
@@ -95,3 +96,19 @@ def gtype(self, value):
                 f"Invalid coordinate system type: {value}, should be "
                 "either 0 for Cartesian or 1 for Geographic."
             )
+
+    def imshow(self):
+        """
+        Image plot of 2D DataArray.
+
+        Examples
+        --------
+
+        >>> from pygmt.datasets import load_earth_relief
+        >>> grid = load_earth_relief()
+        >>> grid.gmt.imshow()
+        """
+        fig = Figure()
+        fig.grdimage(self._obj, frame=True)
+        fig.colorbar()
```

Comment:
> I'm a little on the fence about adding a colorbar. Sometimes the default colorbar looks very bad (e.g. too many black lines).

That's because the original CPT file has too many z-slices. `fig.colorbar(frame=True)` should always be good.